src/ui/molecules/networkGraph/ecs/util.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable @typescript-eslint/prefer-readonly-parameter-types */
```

Comment:
This is unfortunately necessary because the linter requires the parameter `pos: Vector` to be a read only type. But for clarification, and also to narrow the scope, I place the exception at the level of the function.